src/library_sdl.js

```diff
@@ -2394,6 +2394,19 @@ var LibrarySDL = {
       if (!SDL.audioContext) throw 'Web Audio API is not available!';
       SDL.audio.nextPlayTime = 0; // Time in seconds when the next audio block is due to start.
       
+      var sources = [];
```

Comment:
Let's not define new global functions (`SDL.audio.stopSources`) or variables (`var sources`) inside a local function like this, but it would be better to just define them at the same level with `SDL.audio.caller` and `SDL.audio.queueNewAudioData` and the other functions, and rename `sources` to something more descriptive, perhaps `SDL.audio.currentlyPlayingSources`, and `SDL.audio.stopCurrentlyPlayingSources`?